test/pricer_rule/constructor.js

```diff
@@ -113,12 +113,15 @@ contract('PricerRule::constructor', async () => {
         organization,
       } = await PricerRuleUtils.createOrganization(accountProvider);
 
-      const token = accountProvider.get();
+      const tokenDecimals = 10;
```

Comment:
we still work under the assumption that the decimals of UtilityToken (token) and baseToken (OST) is both 18; if they are not the same, the conversion rate needs to absorb this